ontotext-yasgui-web-component/src/services/utils/dialog-util.ts

```diff
@@ -0,0 +1,14 @@
+import {HtmlUtil} from './html-util';
+
+export class DialogUtil {
+  static  preventLeavingDialog(hostElement: HTMLElement, ev: KeyboardEvent) {
```

Comment:
Move this in the HtmlUtil. It has nothing particularly specific related with dialogs.

Reply:
Done